### PR TITLE
Feature/uh2020 load filters from api

### PR DESCRIPTION
**Pull request by jesusdlacrz**
image
image
image

### CODE REVIEW

File: src/components/common/error.tsx
Comment (Edimar-star) [inline src/components/common/error.tsx:1]: well, it's a good job, however fontawesome has that icon, there was no need to install another dependency for the exclamation triangle icon

File: src/services/plpServices.ts
Comment (Edimar-star) [inline src/services/plpServices.ts:1]: although it is good what you do, you could use the filters provided by json server.

**Review by Edimar-star — APPROVED** [review]
Good job!

**Review by RDCelin — APPROVED** [review]
Good job

**Review by Katkat04 — APPROVED** [review]
good job

**Pull request merged**